fabfile.py

```diff
@@ -397,23 +403,23 @@ def run_backup(self, email=False):
         sudo('zip /tmp/{0}-{1}.backup /tmp/odoo-{0}.tar /tmp/{0}.sql'
              .format(self.instance, date))
 
-        #save in instance/backup folder
-        backup = os.path.isdir("backup")
-        if not backup:
-            os.system("mkdir backup")
-        os.system('scp -r applejuice.sunflowerweb.nl:/tmp/{0}.backup backup/{0}.backup'
+        os.system('scp -r applejuice.sunflowerweb.nl:/tmp/{0}* /mnt/volume-fra1-01/sfit-backup'
```

Comment:
Please dont refer explicitly to applejuice.sunflowerweb.nl, but use [fabric get](http://docs.fabfile.org/en/1.14/api/core/operations.html#fabric.operations.get) to get the file.